webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailConfig.js

```diff
@@ -0,0 +1,299 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import LongDateTime from 'foremanReact/components/common/dates/LongDateTime';
+import { startCase } from 'lodash';
+import {
+  BugIcon,
+  SecurityIcon,
+  EnhancementIcon,
+} from '@patternfly/react-icons';
+import {
+  getAnsibleCollections,
+  getContentViewVersions,
+  getDebPackages,
+  getDockerTags,
+  getErrata,
+  getModuleStreams,
+  getPackageGroups,
+  getRepositories,
+  getRPMPackages,
+} from '../../ContentViewDetailActions';
+import {
+  selectAnsibleCollections,
+  selectAnsibleCollectionsStatus,
+  selectCVVersions,
+  selectCVVersionsStatus,
+  selectDebPackages,
+  selectDebPackagesStatus,
+  selectDockerTags,
+  selectDockerTagsStatus,
+  selectErrata,
+  selectErrataStatus,
+  selectModuleStreams,
+  selectModuleStreamsStatus,
+  selectRepositories,
+  selectRepositoriesStatus,
+  selectRPMPackageGroups,
+  selectRPMPackageGroupsStatus,
+  selectRPMPackages,
+  selectRPMPackagesStatus,
+} from '../../ContentViewDetailSelectors';
+import ContentViewVersionRepositoryCell from './ContentViewVersionRepositoryCell';
+
+export const TableType = PropTypes.shape({
+  name: PropTypes.string,
+  getCountKey: PropTypes.func,
+  responseSelector: PropTypes.func,
+  statusSelector: PropTypes.func,
+  autocompleteEndpoint: PropTypes.string,
+  fetchItems: PropTypes.func,
+  columnHeaders:
+    PropTypes.arrayOf(PropTypes.shape({
+      title: PropTypes.string,
+      width: PropTypes.number,
+      getProperty: PropTypes.func,
+    })),
+});
+
+
+export default ({ cvId, versionId }) => [
+  {
+    name: __('Components'),
+    getCountKey: item => item?.component_view_count,
+    responseSelector: state => selectCVVersions(state, cvId),
+    statusSelector: state => selectCVVersionsStatus(state, cvId),
+    autocompleteEndpoint: '',
```

Comment:
Think it's ok to use /katello/api/v2/content_view_versions/auto_complete_search?content_view_id=3 with the search enabled here for consistency and searching based on repository. 

Also, CV Name is appended with version which isn't required. 
![Screenshot from 2021-09-17 13-47-15](https://user-images.githubusercontent.com/21146741/133832281-0de42f09-4f24-401c-86f8-3e259003e672.png)

Reply:
I attempted this, and the components endpoint dies when you attempt to give it a search param.

I fixed the naming issue.